docs/rfcs/20220829-distributed-dynamic-routing.md

```diff
@@ -0,0 +1,224 @@
+- Feature Name: distributed-dynamic-routing
+- Tracking Issue: [#198](https://github.com/CeresDB/ceresdb/issues/198)
```

Comment:
Here is an issue tracking the implementation of this RFC, which is not submitted yet (#198 is an issue requesting this RFC).